src/Router/Router.ts

```diff
@@ -32,6 +30,10 @@ export default class Router {
     };
   }
 
+  public get util (): W3Util {
```

Comment:
Looks like should be removed

Reply:
it is being used to fetch the transaction `getTransaction`